app/sql.js

```diff
@@ -1335,6 +1336,55 @@ function updateToLokiSchemaVersion19(currentVersion, db) {
   console.log(`updateToLokiSchemaVersion${targetVersion}: success!`);
 }
 
+function updateToLokiSchemaVersion20(currentVersion, db) {
+  const targetVersion = 20;
+  // if (currentVersion >= targetVersion) {
+  //   return;
+  // }
+  console.log(`updateToLokiSchemaVersion${targetVersion}: starting...`);
+  db.transaction(() => {
+    db.exec(`
+        UPDATE ${CONVERSATIONS_TABLE} SET
+        json = json_set(json, '$.didApproveMe', 1, '$.isApproved', 1)
+        WHERE type = 'private';
+      `);
+
+    // all closed group admins
+    const closedGroupRows = db
+      .prepare(
+        `
+        SELECT json FROM ${CONVERSATIONS_TABLE} WHERE
+        type = 'group' AND
+        id NOT LIKE 'publicChat:%';
+      `
+      )
+      .all();
+
+    console.warn({ closedGroupRows });
```

Comment:
avoid using console.warn. this should be used for debugging only. You can have console.info if you want to